store/src/commands/bonds.rs

```diff
@@ -0,0 +1,140 @@
+use collect::dto::ValidatorBondRecord;
+use log::info;
+use serde_yaml;
+use std::collections::{HashMap, HashSet};
+use tokio_postgres::types::ToSql;
+use tokio_postgres::NoTls;
+
+use crate::commands::utils::{InsertQueryCombiner, UpdateQueryCombiner};
+
+use super::common::CommonStoreOptions;
+
+const DEFAULT_CHUNK_SIZE: usize = 500;
+
+pub async fn store_bonds(options: CommonStoreOptions) -> anyhow::Result<()> {
```

Comment:
I know where you copied this from but I believe this can be done in a better way (now that i know rust a bit better). Inspiration could be taken from here:
https://github.com/marinade-finance/native-staking/blob/master/bot/src/api/handlers/rewards_metrics.rs#L70

I would approve the PR even without refactor of this section (first please make sure that the business logic is in place). But afterwards please make sure a follow-up PR is made and the `UpdateQueryCombiner` is no longer used.